### PR TITLE
1 table and waiter

### LINKED ISSUE #1: Table and Waiter
- Crud operation for table and Waiter

### PR DESCRIPTION
**Pull request by SaiPranab**
added support for addTable, deleteTable, getAllTable, getTableById
&
addWaiter, deleteWaiter, getAllWaiters, getWaiterById
&
getWaiterAssignedToTable(tableId)
assignTableToWaiter(waiterId, tableId)

### CODE REVIEW

File: backend/src/main/java/com/lafista/backend/table/controller/RestaurantTableController.java
Comment (smruti-banty) [inline backend/src/main/java/com/lafista/backend/table/controller/RestaurantTableController.java:1]: change url from `tables` to `table`

Comment (smruti-banty) [inline backend/src/main/java/com/lafista/backend/table/controller/RestaurantTableController.java:1]: Use `DTO` class to handle request and response

Comment (smruti-banty) [inline backend/src/main/java/com/lafista/backend/table/controller/RestaurantTableController.java:1]: Add method for `getByTableId` and update table name

File: backend/src/main/java/com/lafista/backend/waiter/controller/WaiterController.java
Comment (smruti-banty) [inline backend/src/main/java/com/lafista/backend/waiter/controller/WaiterController.java:1]: Remove response OK

File: backend/src/main/resources/application.properties
Comment (smruti-banty) [inline backend/src/main/resources/application.properties:1]: Remove below properties 

```server.port=1205
spring.jpa.show-sql=true
spring.jpa.hibernate.ddl-auto=update
spring.jpa.properties.hibernate.format_sql=true

**Review by smruti-banty — CHANGES_REQUESTED** [review]
Change the file according to comment and test properly using postman

**Comment by SaiPranab** [comment]
- [ x ] remove response OK
- [ x ] add method for updateTable and getTableById
- [ x ] add method for updateWaiter and getWaiterById
- [ x ] add DTO classes for handling requests
- [ x ] change urls tables to table 
- [ x ] change urls waiters to waiter

**Review by smruti-banty — APPROVED** [review]
LGTM.

**Review by smruti-banty — APPROVED** [review]

**Pull request merged**